aries-site/src/examples/templates/forms/formHelpers.js

```diff
@@ -0,0 +1,62 @@
+export const emailMask = [
```

Comment:
This whole file is great. I like how you separated all of the validations I think we will be using this in many places so much cleaner way to incorporate these helpers.